translations/en.json

```diff
@@ -0,0 +1,8 @@
+{
+  "app": {
+    "name": "Zen Tunes",
```

Comment:
This file seems like it's completely unused. Not sure if it needs to exist, but these strings don't seem to relate to our Rollbar integration at all ;)

Reply:
That's the app description for if we actually wanted to put the integration in the Zendesk app store. Since we're not Rollbar, I'm not sure if we can actually do that. :wink: I've updated it in any case, as I think one of the lines shows up during manual install as well.